packages/frontend-2/nuxt.config.ts

```diff
@@ -50,6 +50,8 @@ export default defineNuxtConfig({
       baseUrl: '',
       mixpanelApiHost: 'UNDEFINED',
       mixpanelTokenId: 'UNDEFINED',
+      survicateWorkspaceKey: 'UNDEFINED',
```

Comment:
1. You should put these in .env.example as well, otherwise devs won't know of these keys and won't be able to set them up locally
2. Don't do 'UNDEFINED' as a default value, it will cause survicate to always be loaded even if the env vars aren't set, but with invalid values ('UNDEFINED') which will just cause erroring requests to survicate